sigs.yaml

```diff
@@ -1098,36 +1098,24 @@ sigs:
       slack: sig-docs
       mailing_list: https://groups.google.com/forum/#!forum/kubernetes-sig-docs
       teams:
+      - name: kubernetes-blog
+        description: Kubernetes blog
       - name: sig-docs-maintainers
         description: Documentation maintainers
       - name: sig-docs-pr-reviews
         description: Documentation PR reviews
       - name: sig-docs-en-owners
         description: English content (default)
-      - name: sig-docs-es-owners
```

Comment:
is there a reason we are removing these teams?

Reply:
@cblecker After thinking about it more, I think it makes sense to keep them. I'll add them back in and fill them out with new localizations.